js/apps/system/_admin/aardvark/APP/aardvark.js

```diff
@@ -1300,39 +1300,75 @@ authRouter.get('/graphs-v2/:name', function (req, res) {
     var sizeCategory;
     var nodeObj;
     var notFoundString = "(attribute not found)";
+
+    const truncate = (str, n) => {
+      return (str.length > n) ? str.slice(0, n-1) + '...' : str;
+    };
     
     const generateNodeObject = (node) => {
       nodeNames[node._id] = true;
+      var label = "";
+      var tooltipText = "";
 
       if (config.nodeLabel) {
-        if (config.nodeLabel.indexOf('.') > -1) {
-          nodeLabel = getAttributeByKey(node, config.nodeLabel);
-          if (nodeLabel === undefined || nodeLabel === '') {
-            nodeLabel = node._id;
+        var nodeLabelArr = config.nodeLabel.trim().split(" ");
+        // in case multiple node labels are given
+        if(nodeLabelArr.length > 1) {
```

Comment:
nit: formatting. Also check all other if statements?
```suggestion
        if (nodeLabelArr.length > 1) {
```